### ISSUE TITLE
Docs incorrectly list full support for chromium browsers

### ISSUE THREAD

**Issue by ConTejas624** [mdn/browser-compat-data#20217]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo

### What specific section or headline is this issue about?

Browser Compatibility

### What information was incorrect, unhelpful, or incomplete?

For Edge and Chrome, the `behavior: "smooth"` option for window.scrollTo() is only supported if the user has enabled the experimental flag `chrome://flags/#smooth-scrolling`. 

### What did you expect to see?

A note saying that the smooth scrolling paramater is supported only if the user has enabled the flag

### Do you have any supporting links, references, or citations?

Comment by Tamas Bolvari on Saeed Hassanvand's answer on this stackoverflow thread https://stackoverflow.com/questions/49636727/why-is-scroll-behaviorsmooth-not-working-but-javascript-window-scroll-smooth-is 

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/window/scrollto`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/window/scrollto/index.md
* Last commit: https://github.com/mdn/content/commit/cc070123f72376faec06e36622c4fc723a75325f
* Document last modified: 2023-04-08T06:57:07.000Z

</details>

**Comment by Elchi3** [comment]
I think version 61 is based on information per https://chromestatus.com/feature/5812155903377408

**Comment by hamishwillee** [comment]
Yes, it looks reasonable. Just doesn't seem to reflect reality.

**Comment by clshortfuse** [comment]
Chrome actually has a bugged implementation of `{behavior: smooth}` for over a year now, so I feel like it should be warned about. 

If you try to smooth scroll two elements it just breaks completely:

* https://bugs.chromium.org/p/chromium/issues/detail?id=1430426
* https://bugs.chromium.org/p/chromium/issues/detail?id=1345593

Since it's been over a year now, it seems like low priority for the Chromium team.

**Comment by caugner** [comment]
> Comment by Tamas Bolvari on Saeed Hassanvand's answer on this stackoverflow thread [stackoverflow.com/questions/49636727/why-is-scroll-behaviorsmooth-not-working-but-javascript-window-scroll-smooth-is](https://stackoverflow.com/questions/49636727/why-is-scroll-behaviorsmooth-not-working-but-javascript-window-scroll-smooth-is)

Note that the StackOverflow comment refers to `scroll-behavior: smooth`, whereas the OP is about `Window.scrollTo()` with `{ behavior: 'smooth' }`.

> * bugs.chromium.org/p/chromium/issues/detail?id=1345593

Note that this bug refers to scroll-snap, and doesn't mention smooth scrolling specifically.

> * bugs.chromium.org/p/chromium/issues/detail?id=1430426

This [bug 40901737](https://issues.chromium.org/issues/40901737) (`Scrolling window or element stops all other scrolls, including scroll-snap`) was marked as duplicate of [bug 40572042](https://issues.chromium.org/issues/40572042) (`scroll-behavior: smooth does not work for multiple elements`) , which in turn was marked as duplicate of [bug 325081538](https://issues.chromium.org/issues/325081538) (`Smooth scrolling works with multiple scrolls`). The implementation (https://github.com/chromium/chromium/commit/73ac2aa3a3cd2e850d153234d32f2252c48fb3ef) was enabled by default via https://github.com/chromium/chromium/commit/dbdf58279ce5708e9c92516f239c9c0f7a2dca1d in Chrome 130 (after being experimental since https://github.com/chromium/chromium/commit/3e0915a828439782292bb62e12a585685bff7da8 in Chrome 128).

tl;dr Chrome 130 started supporting multiple parallel smooth scrolls.

I don't think this justifies marking the `scrollTo()` `behavior` option as partial (note that we don't track subfeatures for the values `smooth`, `instant` and `auto`), although it might be fine to add a note.

/cc @ddbeck @Elchi3 for a second opinion.

**Comment by caugner** [comment]
PS: For some reason, the "Smooth scrolling" flag does not seem to be available on macOS:

<img width="734" height="199" alt="Image" src="https://github.com/user-attachments/assets/cf44c017-982d-4667-b9c3-917c5f171d4e" />

The flag is mentioned in [chrome/browser/flag-metadata.json](https://github.com/chromium/chromium/blame/1275f01594831fbfbd9707337342a355a0221422/chrome/browser/flag-metadata.json#L8887-L8891), but I'm not totally sure what it controls, as `scrollTo()` with `{ behavior: 'smooth' }` works for me in Chrome 150 on macOS.
